components/brave_webtorrent/browser/net/sources.gni

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+brave_components_brave_webtorrent_browser_net_sources = [
```

Comment:
I don't think this is a right way to solve this problem. Basically these helpers are so small that they can be moved to the browser layer (e.g. directly to `/brave/browser/net`. Same works for `content_browser_client_helper.h`

Reply:
If it's allowed to move the files to //brave/browser/net, yes, it's better. I moved them to //brave/browser/net. Thanks.